DigitalLearningSolutions.Data/Services/DiagnosticAssessmentService.cs

```diff
@@ -64,10 +66,13 @@ FROM Sections
                         Customisations.CustomisationID = @customisationId
                         AND Sections.SectionID = @sectionId
                         AND Sections.ArchivedDate IS NULL
-                        AND Sections.DiagAssessPath IS NOT NULL
+                        AND Sections.DiagAssessPath IS NOT NULL
+                        AND CustomisationTutorials.DiagStatus = 1
+                        AND Tutorials.DiagAssessOutOf > 0
+                        AND Tutorials.ArchivedDate IS NULL
```

Comment:
Is the `IsAssessed` check no longer necessary? 🙂

Reply:
It is not checked by the query given in the ticket, so I removed the check. I'm not entirely sure what `IsAssessed` means, I think it might be talking about post learning assessments rather than diagnostic ones but I'm not to sure 🙂

Reply:
Yes you're right, I've just checked the ACs for showing the diagnostic assessment on the section page and IsAssessed doesn't need to be true 🙂